src/utils/componentCategories.ts

```diff
@@ -20,6 +20,10 @@ const categories = {
       "Promo",
     ],
   },
+  deprecated: {
+    title: "Deprecated",
+    components: ["CheckboxOld"],
```

Comment:
This isn't being displayed. Is it suppose to or is this meant to just be a declaration?

Reply:
Michael inserted that when he was reworking `Checkbox`.  We opted to not deprecate the original component, so `CheckboxOld` was never actually created.  However, I think we will eventually want to have a `Deprecated` category, so I left it in there.

Reply:
You already approved this PR, but I removed `CheckboxOld` from the deprecated section.